.github/ISSUE_TEMPLATE/registerAddon.yml

```
@@ -10,7 +10,8 @@ body:
 - type: markdown
   attributes:
     value: |
-      Submit an NVDA add-on to datastore. Provide download and source URL, and indicate if it's a stable add-on or not.
+      Submit an NVDA add-on to datastore.
+      Add-ons should comply with the the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
```

Fix typo (extra 'the'):
```suggestion
      Add-ons should comply with the [NVDA code of conduct](https://github.com/nvaccess/nvda/blob/master/CODE_OF_CONDUCT.md).
```